spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  before do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    @post = Post.create(title: 'Title', text: 'Body', author_id: @user.id)
+  end
+
+  describe 'GET #index' do
+    it 'returns a 200 status code' do
+      subject { get :index, params: { user_id: @user.id } }
+      expect(subject).to have_http_status(200)
+    end
+    it 'render post template' do
+      subject { get :index, params: { user_id: @user.id } }
+      expect(subject).to render_template('index')
+    end
+
+    it 'the response body includes correct placeholder' do
+      subject { get :index, params: { user_id: @user.id } }
+      expect(subject.body).to include('Here is a list of posts for a given user')
+    end
+  end
+
```

Comment:
- [ ] Good job working on the changes requested by the previous reviewer, however, your test is still failing.
![Capture](https://user-images.githubusercontent.com/85095702/188285082-f8357a39-1a81-4090-9d5e-19a30d103204.PNG)

- [ ] Check your models. the validate method should be `validates` and not `Validate`